pkg/synopsysctl/cmd_update.go

```diff
@@ -359,16 +359,43 @@ var updateBlackDuckCmd = &cobra.Command{
 			if err != nil {
 				return err
 			}
+			isSecretUpdated := false
 			for _, v := range secrets {
 				if secret, err := util.GetSecret(kubeClient, namespace, v.Name); err == nil {
 					secret.Data = v.Data
 					secret.StringData = v.StringData
 					if _, err := util.UpdateSecret(kubeClient, namespace, secret); err != nil {
-						return fmt.Errorf("failed to update certificate secret: %+v", err)
+						return fmt.Errorf("failed to update the %s secret due to %+v", v.Name, err)
 					}
 				} else {
 					if _, err := kubeClient.CoreV1().Secrets(namespace).Create(&v); err != nil {
-						return fmt.Errorf("failed to create certificate secret: %+v", err)
+						return fmt.Errorf("failed to create the %s secret due to %+v", v.Name, err)
+					}
+				}
+				isSecretUpdated = true
+			}
+
+			if isSecretUpdated {
```

Comment:
Can we add a comment here that says something like "Restarting the Pods to load the new secret information"